### PR TITLE
NOT FOR MERGE: Avishay/swagger auth type2

### PR DESCRIPTION
**Pull request by avishayt**
How does this look for specifying authentication?  Need to see if we can disable it...

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -16,12 +16,23 @@ consumes:
 produces:
   - application/json
 
+securityDefinitions:
```
Comment (rollandf) [inline swagger.yaml:19]: I was looking at https://swagger.io/docs/specification/authentication/bearer-authentication/

There it is configured like that:
```
components:
  securitySchemes:
    bearerAuth:            # arbitrary name for the security scheme
      type: http
      scheme: bearer
      bearerFormat: JWT 
```

Comment (rollandf) [reply swagger.yaml:19]: Seems I was not looking at 2.0 ..

File: swagger.yaml [hunk 2]
```diff
@@ -224,6 +247,8 @@ paths:
     get:
       tags:
         - installer
+      security:
+        - user: []
```
Comment (avishayt) [inline swagger.yaml:251]: Also agent

File: swagger.yaml [hunk 3]
```diff
@@ -47,6 +58,8 @@ paths:
...
```
Comment (filanov) [inline swagger.yaml:62]: Need to add agent here as well, according to https://github.com/eranco74/assisted-installer/blob/master/src/inventory_client/inventory_client.go#L76

**Pull request closed without merging**